docs/guides/server/keycloak-truststore.adoc

```diff
@@ -2,52 +2,36 @@
 <#import "/templates/kc.adoc" as kc>
 
 <@tmpl.guide
-title="Configuring trusted certificates for outgoing requests"
-summary="How to configure the {project_name} Truststore to communicate with external services through TLS."
+title="Configuring trusted certificates"
+summary="How to configure the {project_name} Truststore to communicate through TLS."
 includedOptions="">
 
-When {project_name} communicates with external services through TLS, it has to validate the remote server’s certificate in order to ensure it is connecting to a trusted server. This is necessary in order to prevent man-in-the-middle attacks. The certificates of these remote server’s or the CA that signed these certificates must be put in a truststore. This truststore is managed by the Keycloak server.
+When {project_name} communicates with external services or has an incoming connection through TLS, it has to validate the remote certificate in order to ensure it is connecting to a trusted server. This is necessary in order to prevent man-in-the-middle attacks.
 
-The truststore is used when connecting securely to identity brokers, LDAP identity providers, when sending emails, and for backchannel communication with client applications. It is also useful
-when you want to change the policy on how host names are verified and trusted by the server.
+The certificates of these clients or servers, or the CA that signed these certificates, must be put in a truststore. This truststore is then configured for use by Keycloak.
 
-By default, a truststore provider is not configured, and any TLS/HTTPS connections fall back to standard Java Truststore configuration. If there is no trust established, then these outgoing requests will fail.
+== Configuring the System Truststore
 
-== Configuring the {project_name} Truststore
+The existing Java default truststore certs will always be trusted.  If you need additional certificates, they can be included in the conf/truststores directory or subdirectories.  The certs may be in PEM files, or PKCS12 files with extension .p12 or .pfx.  If in PKCS12, the certs must be unencrypted - meaning no password is expected.
 
-You can add your truststore configuration by entering this command:
+If you need to use an alternative path, use the --truststore-paths option to specify additional files or directories where PEM or PKCS12 files are located.  Paths are relative to KEYCLOAK_HOME.
 
-<@kc.start parameters="--spi-truststore-file-file=myTrustStore.jks --spi-truststore-file-password=password --spi-truststore-file-hostname-verification-policy=ANY"/>
+After all applicable certs are included, the truststore will be used as the system default truststore via the javax.net.ssl properties, and as the default for internal usage within {project_name}.  
```

Comment:
Thinking about it more, it's an implementation details (that we use the system properties) with no impact on users (as we count with the possibility the properties are set by the user). I'd probably remove this from the docs.

Reply:
I think it bears mentioning.  The previous spi mechanism was fully separate from the system properties, users could have been using both.  It should be clear that this option supercedes the existing system properties.